usr/share/rear/lib/global-functions.sh

```diff
@@ -169,9 +171,19 @@ umount_url() {
                 return 0
             fi
             ;;
-        (sshfs)
-            umount_cmd="fusermount -u $mountpoint"
-            ;;
+	    (sshfs)
+	        umount_cmd="fusermount -u $mountpoint"
+	    ;;
+	    (davfs)
+	        umount_cmd="umount $mountpoint"
```

Comment:
I am afraid that `umount_cmd` is not used anywhere.

Reply:
See
https://github.com/rear/rear/commit/20359a987662cc0c3fcfa52d62d1feac7cd55850#r51320967